Daytime-Operations/Auxiliary-Telescope/AT-Daytime-Checkout/Daytime Checkout SAL Scripts.rst

```diff
@@ -0,0 +1,93 @@
+and alert the daytime staff to any operational issues before going on-sky. These tests are designed to be run during the 
+daytime with an Observing Specialist or other personnel available to verify their results. 
+
+The scripts are intended to be run in order, but offer some flexibility in what state to leave the Auxiliary Telescope (e.g. 
+ready to take calibrations, ready to observe at night, or simply parked and shutdown). They can also be run individually if a 
+single component is undergoing maintenence and requires a check of basic functionality. 
+
+None of the scripts require configuration. 
+
+.. Important::
+
+    The daytime checkout scripts involve Telescope, Dome, and Instrument component movement. You must announce that you are running
+    the checks on the #summit-announce slack channel before execution. 
+
+The order of script execution is nominally:
+
+* enable_latiss - Can be skipped if LATISS enabled 
+* latiss_checkout
+* enable_atcs - Can be skipped if ATCS already enabled
+* atpnuematics_checkout
+* telescope_and_dome_checkout
+* slew_and_take_image_checkout
+* prepare_for/flats - Can be substituted for other prepare_for scripts to leave AuxTel in different state
+
+*enable_latiss*
+===============
+
+Bring LATISS to an enabled state to prepare for checkout. 
+
+*latiss_checkout*
+=================
+
+This script will take a bias and engineering frame with latiss and ensure that they are ingested by the OODS.
+
+Check that the bias pops up in the monitor (https://roundtable.lsst.codes/rubintv/summit/auxtel/monitor_current)
+If it fails, let Patrick or Merlin know
+
+You can check that it ingested at USDF with this link: ******* <b>Add when available<b>
+
+*enable_atcs*
+=============
+
+Bring the ATCS into an enabled state to prepare for checkout. 
+
+*pnuematics_checkout*
+=====================
+
+This script will raise and lower the mirror, exercising the air pressure system. It will also open and close the mirror covers and vents. 
+
+*telescope_and_dome_checkout*
+=============================
+
+This script will move the telescope and dome. You MUST announce on the #summit-announce slack channel before executing this script
+as it results in movement. 
+
+*slew_and_take_image*
+=============================
+
+This script will slew the telescope and dome to two different targets, pausing at each target to take a verification image slew_and_take_image
+perform sidereal tracking for 3 minutes. The mirror covers remain closed during the duration of the script. At the end of the script the Telescope and Dome are parked. 
+
+*prepare_for/flats*
+===================
+This is the default end-of-checkout script. It will leave the telescope in position to start taking afternoon calibraitons. If you are not sure how you want to leave the telescope,
+consider running this script as it will perform a few more useful checks of the system. 
```

Comment:
What kind of checks of the system?

Reply:
Good question, I'm actually not sure what specifically it does in addition to the daytime checks we've already done. Maybe I'll reword this.